torch/_inductor/codegen/common.py

```diff
@@ -569,7 +570,11 @@ def bitwise_right_shift(x, y):
     @staticmethod
     def remainder(a, b):
         r = ops.mod(a, b)
-        return ops.where(f"(({r} != 0) & (({r} < 0) != ({b} < 0)))", ops.add(r, b), r)
+        cond = ops.and_(
+            ops.ne(r, ops.constant(0, torch.int32)),
+            ops.ne(ops.signbit(r), ops.signbit(b)),
```

Comment:
Why change to signbit here?

Reply:
It's just more descriptive of what's going on here.

Reply:
It's also a much more complicated compound op instead of a divice primitive though.

Reply:
We do lower it to `x < 0` in triton for integer dtypes and to signbit otherwise in triton and C++. I figured that it'd be pretty much equivalent to an `&` by a constant, and I figured it'd be equivalent to `x < 0`.

Anyway, not a big deal, reverted.

Reply:
Oh, I was mixing this up with `ops.sign` . This is lowered as `libdevice.signbit` which I would expect to have a 1 cycle latency, so yeah not an issue.